apps/oc_chef_authz/src/oc_chef_cookbook_artifact_version.erl

```diff
@@ -80,17 +80,15 @@ update_query() ->
     erlang:error(not_supported).
 
 delete_query() ->
-    %% TODO: do we need this?
-    erlang:error(not_supported).
+    delete_cookbook_artifact_version_by_id.
 
 find_query() ->
     find_cookbook_artifact_version_by_org_name_identifier.
 
 list_query() ->
-    list_cookbook_artifact_versions_by_org_id.
+    erlang:error(not_supported).
```

Comment:
This might have more value for manage, but, at this stage, there's nothing driving it.